### PR TITLE
Bslib fixes

### PR DESCRIPTION
**Pull request by TwitchBronBron**
Adds:
 - support for both npm aliased bslib AND non-aliased:
     - `ropm install bslib@npm:@rokucommunity/bslib`
     - `ropm install @rokucommunity/bslib`

Fixes:
 - bug with npm alias of bslib for node version 10
 - infinite loop when rewriting the bslib file during transpile.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -271,6 +271,10 @@ bslib is actually published to npm under the name [@rokucommunity/bslib](https:/
 ropm install bslib@npm:@rokucommunity/bslib
 ```
 
+If you're using a version of NodeJS that doesn't support npm aliases, then you can also simply install @rokucommunity/bslib directly without the alias, and BrighterScript will recognize this as well.
```
Comment (elsassph) [inline README.md:274]: Why even recommending the alias-based approach if the simple install works as well?

Comment (elsassph) [reply README.md:274]: Oh ok, then maybe you should say that "If you use Node 12+" it is "recommended" to install using the alias.

Comment (elsassph) [reply README.md:274]: PS: Generally whitespace in the readme is a bit random.
Sometimes there is a blank line around titles, sometimes not.
My OCD, it is hurt.

**Review by elsassph — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (TwitchBronBron) [reply README.md]: @elsassph I tweaked the readme to be a little more clear. Do you like that any better? 

I really like using `bslib_` as a prefix. `rokucommunity_bslib_` is really verbose, and kind of gross to look at (even though yes, I know the transpiled code isn't exactly pretty anyway...). I could be swayed to remove support for the `bslib` alias entirely if @georgejecook and @chrisdp  and @triwav also agree it's not a big deal to keep the `bslib_` shorter alias. 

I also added some whitespace before the headers. I've been sloppy with that, sorry!

Comment (triwav) [reply README.md]: I'd prefer `bslib_` as well. If I'm understanding correctly they have to install it themselves vs having ropm do it for them @TwitchBronBron?

Comment (georgejecook) [reply README.md]: I say keep the short alias

Comment (TwitchBronBron) [reply README.md]: Ok cool. I'll keep the logic as-is, (look for the short alias first, but support the full version if present to support older versions of node). Thanks!

**Pull request merged**